tests/minting.rs

```diff
@@ -0,0 +1,296 @@
+use gear_lib::non_fungible_token::token::{Token as NFToken, TokenMetadata};
+
+pub mod utils;
+use utils::*;
```

Comment:
The same about Rust idiom.
```suggestion
```

Reply:
Not sure if it'd be better because adding `utils::` to every constant that imported by `utils::*` won't be very good. I can try to do something like a prelude module.